pom.xml

```diff
@@ -53,12 +52,6 @@
             <artifactId>syslog-java-client</artifactId>
             <version>1.1.7</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.code.findbugs</groupId>
-            <artifactId>jsr305</artifactId>
-            <version>3.0.2</version>
-            <optional>true</optional>
-        </dependency>
```

Comment:
No longer necessary dependency: have migrated from JSR 305 to SpotBugs annotations below.